server.js

```diff
@@ -42,22 +43,32 @@ var serveBrowserified = function(file, standaloneName) {
             res.send(body);
         });
     };
-};
-
-app.get("/katex.js", serveBrowserified("katex", "katex"));
-app.use("/test/jasmine",
-    express["static"](
-        path.dirname(
-            require.resolve("jasmine-core/lib/jasmine-core/jasmine.js")
-        )
-    )
-);
-app.get("/test/katex-spec.js", serveBrowserified("test/*[Ss]pec.js"));
-app.get("/contrib/auto-render/auto-render.js",
-        serveBrowserified("contrib/auto-render/auto-render",
-                          "renderMathInElement"));
-
-app.get("/katex.css", function(req, res, next) {
+}
+
+function twoBrowserified(url, file, standaloneName) {
+    app.get(url, serveBrowserified(file, standaloneName, false));
+    app.get("/babel" + url, serveBrowserified(file, standaloneName, true));
```

Comment:
I tried this out and there's a noticeable lag... thanks for setting this up a separate route.